source/style-guide/style/product-names-version-numbers.txt

```diff
@@ -4,44 +4,85 @@
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
 
--  In some cases, you can refer to the product generically after using
-   the product name. For example, after you introduce the Cloud
-   Monitoring Agent, you can refer to simply *the agent*.
+   * - Guideline
+     - Example
 
--  Don't capitalize an item that a user creates through a MongoDB
-   service. For example, users use the Cloud Servers service to create
-   a *server*, not a *Server*, and they use the Cloud Load Balancer
-   service to create a *load balancer*, not a *Load Balancer*.
+   * - Spell out and properly capitalize MongoDB product and service
+       names.
+     -
+       - MongoDB Atlas
+       - MongoDB Compass
 
--  Don't abbreviate MongoDB names, unless the abbreviation has been
-   approved by the Legal and Marketing departments. For example, never
-   abbreviate Cloud Block Storage as CBS.
+   * - On the first mention, use the full MongoDB product name.
+       Subsequent mentions may use the product name without MongoDB.
 
--  For API documentation, the version number in the documentation
-   should match the version number of the software. The combination of
-   the API version number and the publication date identify the
-   document version.
+     - .. list-table::
+          :widths: 50 50
+          :header-rows: 1
 
-When using third-party company and product names, use the name as it's
-used by the third-party. For a list of commonly used third-party names,
-see :ref:`third-party-names-and-trademarks`.
+          * - First mention
+            - Subsequent mentions
+
+          * - MongoDB Atlas
+            - Atlas
+
+          * - MongoDB BI Connector
+            - BI Connector
+
+          * - MongoDB Cloud Manager
+            - Cloud Manager
+
+          * - MongoDB Ops Manager
+            - Ops Manager
+
+          * - MongoDB Cloud Manager and Ops Manager
+            - Cloud Manager and Ops Manager
+
+          * - MongoDB Compass
+            - Compass
+
+          * - MongoDB Realm
+            - Realm
 
-When referring to an OpenStack service, use the actual service name,
-and provide the project name in parentheses. For example, use OpenStack
-Block Storage (Cinder). On subsequent references, use the service name
-instead of the project name, unless you need to use project names to
-differentiate between two versions of one service. See the `OpenStack
-documentation conventions
-<http://docs.openstack.org/contributor-guide/writing-style/openstack-components.html>`__
-for service and project names.
+          * - MongoDB Spark Connector
+            - Spark Connector
+
+   * - You can refer to some products generically after using
+       the product name.
+
+     - You can refer to the MongoDB Agent as simply *the agent*.
+
+   * - Don't capitalize an item that a user creates through a MongoDB
+       service.
+     - Users leverage Atlas to create a:
+
+         - *database*, not a *Database*.
```

Comment:
These appear to render as bullets in a block quote. I think the syntax should match that used above for the sub-bullets under: " Spell out and properly capitalize MongoDB product and service service..."

Reply:
Fixed.